tests/scripts/all.sh

```diff
@@ -1383,6 +1383,21 @@ component_build_default_make_gcc_and_cxx () {
     make TEST_CPP=1
 }
 
+component_build_module_alt () {
+    msg "build: MBEDTLS_XXX_ALT" # ~30s
+    scripts/config.py full
+    # Disable options that are incompatible with some ALT implementation.
```

Comment:
Perhaps we should have comments explaining the incompatibility. Debug is not obvious to me, and it's not obvious to me either why AESNI is incompatible while PADLOCK isn't.

Reply:
I determined this list by enabling everything and then disabling the minimum I could to get the library to compile. The code of `padlock.c` is only actually enabled on 32-bit x86, and this component does a default-architecture build which in practice means 64-bit x86. I'll exclude `MBEDTLS_PADLOCK_C` anyway for robustness.